tests/unit/test.program.js

```diff
@@ -6,7 +6,11 @@ import sinon from 'sinon';
 import {assert} from 'chai';
 import {spy} from 'sinon';
 
-import {defaultVersionGetter, main, Program} from '../../src/program';
+import {
+  defaultVersionGetter,
+  main,
+  Program,
```

Comment:
Any reason why you split these into multiple lines? Since the previous statement was less than 80 characters, you don't need to split it.